cypress/e2e/swap/errors.test.ts

```diff
@@ -116,4 +116,12 @@ describe('Swap errors', () => {
       getBalance(DAI).should('be.closeTo', initialBalance + 200, 1)
     })
   })
+
+  it('insufficient liquidity', () => {
+    cy.visit(`/swap?inputCurrency=${USDC_MAINNET.address}&outputCurrency=${DAI.address}`)
+    // This is assuming there is not > 100 Trillion Dai in a pool
+    cy.get('#swap-currency-output .token-amount-input').type('100000000000000').should('have.value', '100000000000000')
+    cy.contains('Insufficient liquidity for this trade.')
```

Comment:
If it's known that our API takes a long time to resolve this, you should explicitly add a timeout here to account for that. Otherwise this test will be flaky.

If you do extend the timeout, leave a comment explaining why:

```suggestion
    // The API takes X seconds to return a 404, so we extend the timeout.
    cy.contains('Insufficient liquidity for this trade.', { timeout: 48000 ))
```